lib/memoize-string-transformer.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+const stringTransformerCache = Symbol('stringTransformerCache');
+let methodId = 0;
+
+// Helper function that converts class methods into memoized versions
+// of themselves with a fixed-size LRU cache.
+module.exports = function memoizeStringTransformerMethod (cacheSize, fn) {
+  const id = methodId++;
```

Comment:
Can't this be avoided with a different cache variable local to each method? Something like:

function memoizeStringTransformerMethod (cacheSize, fn) {
    const cache = new Map();
    return function (str) {
        if (cache.has(str)) {
            ...
        }
        ...
    }
}

Reply:
Yes. :facepalm: This was nice for debugging, but you’re right, of course this isn’t necessary anymore.